### ISSUE TITLE
Make sure tests do not depend on organization viewer permission

### ISSUE THREAD

**Issue by morgante** [terraform-google-modules/terraform-google-project-factory#216]

**Comment by morgante** [comment]
To ensure that #20 doesn't get repeated.

**Comment by paulpalamarchuk** [comment]
I've made some test, and discovered that core module depends on org-level permissions:
```
       Error: Error reading Organization Not Found : 684124036889: googleapi: Error 403: The caller does not have permission, forbidden

         on ../../../modules/gsuite_group/main.tf line 26, in data "google_organization" "org":
         26: data "google_organization" "org" {
```
That happens because of calling [`gsuite_group` module](https://github.com/terraform-google-modules/terraform-google-project-factory/blob/master/main.tf#L21) from core-module. That has [data org resource](https://github.com/terraform-google-modules/terraform-google-project-factory/blob/master/modules/gsuite_group/main.tf#L18)

**Comment by imrannayer** [comment]
Fixed in #320
